substrate/client/keystore/src/local.rs

```diff
@@ -369,7 +398,20 @@ impl Keystore for LocalKeystore {
 			key_type: KeyTypeId,
 			seed: Option<&str>,
 		) -> std::result::Result<ecdsa_bls381::Public, TraitError> {
-			self.generate_new::<ecdsa_bls381::Pair>(key_type, seed)
+			let pubkey = self.generate_new::<ecdsa_bls381::Pair>(key_type, seed)?;
+
+			let s = self
+				.0
+				.read()
+				.additional
+				.get(&(key_type, pubkey.to_vec()))
+				.map(|s| s.to_string())
+				.expect("Can retrieve seed");
+
```

Comment:
```suggestion

// This is done to give the keystore access to individul keys, this is necessay to avoid unnecessary host functions for paired keys and re-use host functions implemented for each element of the pair.
```